File: C++/Product.cpp
#include<iostream>
#include<string>

using namespace std;

class Product{


    private :
    int idProduct;
    int price;


    public :
    Product(){

    }

    Product(int id, int p){
        idProduct = id;
        price = p;
    }

    void setIdProduct(int id){
        idProduct = id;
    }

    void setPrice(int p){
        price = p;
    }

    int getIdProduct(){
        return idProduct;
    }

    int getPrice(){
        return price;
    }

    

    ~Product(){

    }
};

File: Python/Product.py
class Product :


    def __init__(self, idProduct= 0, price = 0):
        self.idProduct = idProduct
        self.price = price
        
    def set_IdProduct(self, idProduct):
        self.idProduct = idProduct    

    def set_price(self, price):
        self.price = price

    
    def get_idProduct(self):
        return self.idProduct

    def get_price(self):
        return self.price

 
       
